eng/targets/VisualStudio.targets

```diff
@@ -93,12 +85,12 @@
       <PkgDefBindingRedirect Include="@(_RuntimeAssetsWithMetadata->'%(Path)')" Condition="'%(_RuntimeAssetsWithMetadata.PkgDefEntry)' == 'BindingRedirect'" />
       <PkgDefCodeBase Include="@(_RuntimeAssetsWithMetadata->'%(Path)')" Condition="'%(_RuntimeAssetsWithMetadata.PkgDefEntry)' == 'CodeBase'" />
 
-      <!-- Check that NuGetPackageToIncludeInVsix doesn't list unreferenced package -->
-      <_UnknownAssets Include="@(NuGetPackageToIncludeInVsix)" />
+      <!-- Check that _NuGetPackageToIncludeInVsix doesn't list unreferenced package -->
+      <_UnknownAssets Include="@(_NuGetPackageToIncludeInVsix)" Condition="'%(_NuGetPackageToIncludeInVsix.IncludeInVsix)' == 'true'" />
```

Comment:
_NuGetPackageToIncludeInVsix is already filtered to IncludeInVsix = true, why the condition here a second time?

Reply:
It's either IncludeInVsix or PkgDefEntry:

`Condition="'%(PackageReference.IncludeInVsix)' == 'true' or '%(PackageReference.PkgDefEntry)' != ''"`